DFE-Digital/early-careers-framework
# Fix incorrect template name on IC material pages

### Context
https://sentry.io/organizations/dfe-bat/issues/2482500190/?project=5748989&referrer=RegressionActivityEmail

### Changes proposed in this pull request
Fix the bug by fixing the typo.
Also add tests to make sure it is less likely to happen in future.

### How can I view this in a review app?
1. Sign in as `school-leader@example.com`
2. Select EDT as your CIP
3. Go to the cohort page
4. Click on the link to EDT's materials at the bottom of the page
5. It should show induction materials for EDT page rather than an error page
